scripts/stubsabot.py

```diff
@@ -183,6 +183,70 @@ def get_updated_version_spec(spec: str, version: packaging.version.Version) -> s
     return _check_spec(".".join(rounded_version) + ".*", version)
 
 
+@functools.cache
+def get_github_api_headers() -> dict[str, str]:
```

Comment:
nit: if we're caching, should type as Mapping

Reply:
Ah, good point!